src/components/InlineNotification/InlineNotification.module.css

```diff
@@ -0,0 +1,104 @@
+@import '../../design-tokens/mixins.css';
+
+/*------------------------------------*\
+    # INLINE NOTIFICATION
+\*------------------------------------*/
+
+/**
+ * A messaging element that is used inline.
+ */
+.inline-notification {
+  display: inline-flex;
+  padding: var(--eds-size-half);
+  border: var(--eds-theme-border-width) solid;
+  border-radius: var(--eds-theme-border-radius);
+
+  background-color: var(--eds-theme-color-background-neutral-default);
+
+  @mixin eds-theme-typography-body-text-lg;
```

Comment:
what is this styling for if the notification text has its own styling?

Reply:
Ack, remnants of a previous commit, will remove